### PR TITLE
K8SPSMDB-388: control how often oplogs are uploaded to S3

### PR DESCRIPTION
**Pull request by defbin**
[K8SPSMDB-388](https://jira.percona.com/browse/K8SPSMDB-388) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| one-pod | passed |
| storage | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| demand-backup | passed |
| limits | passed |
| monitoring-2-0 | passed |
| scaling | passed |
| arbiter | passed |
| scheduled-backup | passed |
| security-context | passed |
| self-healing-chaos | passed |
| rs-shard-migration | passed |
| service-per-pod | passed |
| demand-backup-sharded | failed |
| liveness | passed |
| operator-self-healing | passed |
| smart-update | passed |
| operator-self-healing-chaos | passed |
| version-service | passed |
| users | passed |
| data-sharded | passed |
| non-voting | passed |
| upgrade | passed |
| upgrade-sharded | passed |
| pitr | passed |
| pitr-sharded | failed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/787/commits/249b3ee453594010da9d634f4bfcffbf33571ce4
image: `perconalab/percona-server-mongodb-operator:PR-787-249b3ee4`

**Review by egegunes — DISMISSED** [review]

**Review by qJkee — DISMISSED** [review]

File: deploy/cr.yaml [hunk 1]
```diff
@@ -419,6 +419,7 @@ spec:
 #          prefix: ""
     pitr:
       enabled: false
+#       oplogSpanMin: 10
```
Comment (delgod) [inline deploy/cr.yaml:422]: please check tabulation - line should be in the right place if you remove one symbol `#`
```suggestion
#      oplogSpanMin: 10
```

**Review by delgod — APPROVED** [review]

**Review by delgod — DISMISSED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| storage | passed |
| init-deploy | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| limits | passed |
| scaling | passed |
| arbiter | passed |
| scheduled-backup | passed |
| security-context | passed |
| self-healing-chaos | passed |
| service-per-pod | passed |
| rs-shard-migration | passed |
| demand-backup-sharded | failed |
| liveness | passed |
| operator-self-healing | passed |
| smart-update | passed |
| operator-self-healing-chaos | passed |
| version-service | passed |
| users | passed |
| data-sharded | passed |
| non-voting | passed |
| upgrade | passed |
| upgrade-sharded | passed |
| pitr | passed |
| pitr-sharded | failed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/787/commits/894053e7fb25ad9dc7ad1395d3edadd16f7926bd
image: `perconalab/percona-server-mongodb-operator:PR-787-894053e7`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| one-pod | passed |
| init-deploy | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| limits | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| scaling | passed |
| arbiter | passed |
| security-context | passed |
| self-healing-chaos | passed |
| scheduled-backup | passed |
| rs-shard-migration | passed |
| service-per-pod | passed |
| demand-backup-sharded | failed |
| liveness | passed |
| operator-self-healing | passed |
| smart-update | passed |
| operator-self-healing-chaos | passed |
| version-service | passed |
| users | passed |
| upgrade | passed |
| upgrade-sharded | passed |
| data-sharded | passed |
| pitr | passed |
| non-voting | passed |
| pitr-sharded | failed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/787/commits/7f878bec691ffbc6f3470a2ce6f085fc97ffe0ed
image: `perconalab/percona-server-mongodb-operator:PR-787-7f878bec`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| init-deploy | passed |
| storage | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| limits | passed |
| demand-backup | passed |
| monitoring-2-0 | passed |
| scaling | passed |
| arbiter | passed |
| scheduled-backup | passed |
| security-context | passed |
| self-healing-chaos | passed |
| rs-shard-migration | passed |
| service-per-pod | passed |
| demand-backup-sharded | failed |
| liveness | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| upgrade | passed |
| smart-update | passed |
| upgrade-sharded | passed |
| version-service | passed |
| pitr | failed |
| users | passed |
| data-sharded | passed |
| non-voting | passed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/787/commits/d1b26079a5ad7c90f797f72ce3b5fde62734cf77
image: `perconalab/percona-server-mongodb-operator:PR-787-d1b26079`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| init-deploy | passed |
| storage | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| self-healing-chaos | passed |
| limits | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| scaling | passed |
| arbiter | passed |
| scheduled-backup | passed |
| security-context | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| rs-shard-migration | passed |
| service-per-pod | passed |
| demand-backup-sharded | failed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| users | passed |
| upgrade | passed |
| upgrade-sharded | passed |
| data-sharded | passed |
| pitr | passed |
| non-voting | passed |
| pitr-sharded | failed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/787/commits/cd489d5324b61c7e68c0bd881d55f6d242819a7d
image: `perconalab/percona-server-mongodb-operator:PR-787-cd489d53`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| one-pod | passed |
| storage | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| self-healing-chaos | passed |
| limits | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| scaling | passed |
| arbiter | passed |
| security-context | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| rs-shard-migration | passed |
| scheduled-backup | passed |
| service-per-pod | passed |
| liveness | passed |
| demand-backup-sharded | failed |
| smart-update | passed |
| version-service | passed |
| users | passed |
| upgrade | passed |
| data-sharded | passed |
| upgrade-sharded | passed |
| non-voting | passed |
| pitr | passed |
| pitr-sharded | failed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/787/commits/87d7ccad5eeac22dd34a5d1b230ea1f1ecacd49a
image: `perconalab/percona-server-mongodb-operator:PR-787-87d7ccad`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| storage | passed |
| monitoring-2-0 | passed |
| self-healing | passed |
| arbiter | passed |
| self-healing-chaos | passed |
| operator-self-healing | passed |
| service-per-pod | passed |
| operator-self-healing-chaos | passed |
| liveness | passed |
| init-deploy | passed |
| smart-update | passed |
| limits | passed |
| version-service | passed |
| scaling | passed |
| users | passed |
| security-context | passed |
| data-sharded | passed |
| rs-shard-migration | passed |
| non-voting | passed |
| upgrade-consistency | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| demand-backup-sharded | failed |
| upgrade | passed |
| upgrade-sharded | passed |
| pitr | passed |
| pitr-sharded | passed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/787/commits/87d7ccad5eeac22dd34a5d1b230ea1f1ecacd49a
image: `perconalab/percona-server-mongodb-operator:PR-787-87d7ccad`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| one-pod | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| self-healing-chaos | passed |
| demand-backup | passed |
| monitoring-2-0 | passed |
| scheduled-backup | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| demand-backup-sharded | failed |
| arbiter | passed |
| service-per-pod | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| upgrade | passed |
| users | passed |
| upgrade-sharded | passed |
| pitr | passed |
| data-sharded | passed |
| non-voting | passed |
| pitr-sharded | failed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/787/commits/ec68947549d9bd21fb5df2177bd5c1b8b898254e
image: `perconalab/percona-server-mongodb-operator:PR-787-ec689475`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| init-deploy | passed |
| one-pod | passed |
| upgrade-consistency | passed |
| self-healing | passed |
| self-healing-chaos | passed |
| demand-backup | passed |
| limits | passed |
| monitoring-2-0 | passed |
| scaling | passed |
| scheduled-backup | passed |
| arbiter | passed |
| security-context | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| rs-shard-migration | passed |
| demand-backup-sharded | passed |
| service-per-pod | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| upgrade | passed |
| upgrade-sharded | passed |
| users | passed |
| pitr | passed |
| data-sharded | passed |
| non-voting | passed |
| pitr-sharded | failed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/787/commits/35ba50b58dc085b63c6f76f04a76203accba128c
image: `perconalab/percona-server-mongodb-operator:PR-787-35ba50b5`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| limits | passed |
| scaling | passed |
| security-context | passed |
| rs-shard-migration | passed |
| one-pod | passed |
| monitoring-2-0 | passed |
| arbiter | passed |
| service-per-pod | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| storage | passed |
| users | passed |
| self-healing | passed |
| data-sharded | passed |
| self-healing-chaos | passed |
| non-voting | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| upgrade-consistency | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| demand-backup-sharded | passed |
| upgrade | passed |
| upgrade-sharded | passed |
| pitr | passed |
| pitr-sharded | passed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/787/commits/35ba50b58dc085b63c6f76f04a76203accba128c
image: `perconalab/percona-server-mongodb-operator:PR-787-35ba50b5`

**Review by egegunes — APPROVED** [review]

**Review by hors — APPROVED** [review]

**Review by delgod — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| storage | passed |
| limits | passed |
| self-healing | passed |
| scaling | passed |
| self-healing-chaos | passed |
| security-context | passed |
| operator-self-healing | passed |
| rs-shard-migration | passed |
| operator-self-healing-chaos | passed |
| one-pod | passed |
| monitoring-2-0 | passed |
| arbiter | passed |
| service-per-pod | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| users | passed |
| data-sharded | passed |
| non-voting | passed |
| upgrade-consistency | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| demand-backup-sharded | passed |
| upgrade | passed |
| upgrade-sharded | passed |
| pitr | passed |
| pitr-sharded | passed |

commit: https://github.com/percona/percona-server-mongodb-operator/pull/787/commits/a5f2724020bca6637dd06c0a02240e92320d2879
image: `perconalab/percona-server-mongodb-operator:PR-787-a5f27240`

**Pull request merged**